### DISCUSSION TITLE
ytdl is installed but don't have the right permissions.Please execute command sudo chmod 755 /srv/nextcloud/apps/ncdownloader/lib/Ytdl/../../bin/yt-dlp

### DISCUSSION THREAD

**Discussion by blackcrack** [shiningw/ncdownloader#83] [Q&A]
Hi,

>ytdl is installed but don't have the right permissions.Please execute command sudo chmod 755 /srv/nextcloud/apps/ncdownloader/lib/Ytdl/../../bin/yt-dlp

is announcing the non executable binary for "apache" or the Web-Server, 
i have an own fedoraserver at the basement with nextcloud and this "Issue"

i want give you the info at hand @shiningw , you should add a chmod rule at Nextcloud "chmod"-Checker for these yt-dlp binary,
because i have chmod it to 755 and 777 yt-dlp and after every reload of the nextcloud is this binary again by read only on 
```  
drwxr-xr-x.  2 apache apache     4096 21. Aug 11:02 .
drwxr-xr-x. 13 apache apache     4096 21. Aug 11:02 ..
-rw-r--r--.  1 apache apache 10421536 21. Aug 11:02 aria2c
-rwxr--r--.  1 apache apache  2454824 21. Aug 11:02 yt-dlp
[root@blackysgate /srv/nextcloud/apps/ncdownloader/bin]# 
```  
.. i guess, there is a writing chmod-rule anywhere by Nextcloud or something and must change to 755 to have the possible for execute by webserver"user" .

best  
Blacky

**Comment by blacktemplar** [comment]
I have this problem too, any solutions to this?
